src/containers/MathObjects/components/StatusSymbol/components/ColorSquare.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropType from 'prop-types'
+import Pointable from 'react-pointable'
+
+const Color = styled.div`
+  background-color: ${props => props.color};
+  border-radius: 5px;
+  width: 30px;
+  height: 30px;
+  border: ${props => props.hover ? '1px solid black' : 'none'};
+  cursor: pointer;
+`
+
+export default class ColorSquare extends React.PureComponent {
```

Comment:
If all you're trying to achieve with this component is conditional styling based on hover, you can just do that with Styled Components:

```
export default styled.div`
  background-color: ${props => props.color};
  border-radius: 5px;
  width: 30px;
  height: 30px;
  cursor: pointer;
  &:hover {
    border: 1px solid black
  }
`
```
then you can pass `onPickColor` directly to this `styled.div`, since all `divs` have an `onClick` property.

BTW: In styled-components, `& ... { ... }`  is how you nest rules: https://www.styled-components.com/docs/faqs

Reply:
Oh: styling can always be tweaked later, but I found the black border a bit hard to see. Maybe it would be better to slightly increase the size of the ColorSquare on hover.